Shared/lib/Dancer/Plugin/Database/Core/Handle.pm

```diff
@@ -438,7 +408,50 @@ sub _generate_sql {
     return ($sql, @bind_params);
 }
 
-sub _get_where_sql {
+sub _generate_where_clauses {
```

Comment:
Minor, but important: if we're making this something that external code can reasonably rely on (which is the idea here), then the leading underscore should go, as it's no longer private :)